src/models/device.cpp

```diff
@@ -59,6 +59,8 @@ QString Device::currentDeviceReport() {
   return buffer;
 }
 
+QString Device::uniqueDeviceId() { return QSysInfo::machineUniqueId(); }
```

Comment:
Just a note: We need a followup bug here for android as machineUniqueId is not implemented for it: 
https://bugreports.qt.io/browse/QTBUG-72684

Reply:
Stupid question: what happens on android in this case then?

Reply:
I READ THE DOCS! SHOCKING! The answer is: "If no unique ID could be determined, this function returns an empty byte array."